### DISCUSSION TITLE
enzyme that cuts K* but not RP

### DISCUSSION THREAD

**Discussion by vyaspujari** [vdemichev/DiaNN#941] [General]
I digest with Trypsin and LysC enzyme mix. When analyzing DDA data, I specify the enzyme as being able to cut K* and R* but not RP.

I am trying to find a similar digest specification in DIA-NN but am not having any success. For example,

Trypsin/P gives the same results as --cut K*,R*

but using 

--cut K*
--cut R*

yields a very low number of identified peptides. I am not sure why.

Thus. my attempt of trying to use 

--cut R*,!*P"
--cut K*
does not seem to work. --cut R*,!*P", K* and --cut K*,R*,!*P" also do not seem to work

If someone could tell me how to specify a digest that cuts K* and R* but not RP (trypsin/lysc mix digest) I would appreciate it. It seems though, that even though many people do this type of digest, they leave the default digestion as Trypsin/P. Wouldn't this classify peptides containing RP as missed cleavages? Would this affect the number of quantified peptides?

**Comment by vdemichev** [comment]
--cut command must be supplied only once.
The easiest is to use Trypsin/P. Trypsin does cut before prolines sometimes, and when it does not, few false peptides generated will not really affect DIA-NN search. Yes, RP will be a missed cleavage. Default settings already include 1 missed cleavage allowed, so that's fine. Can increase that to 2 and see if you get any benefit (for non-phospho analyses probably no benefit).
